test/deepSearch.test.js

```diff
@@ -23,25 +23,22 @@ const testData = {
 describe("deepSearch", () => {
   test("Deep search test", () => {
     expect(
-      deepSearch(testData, item => item.type === "string").toEqual([
-        { type: "string", value: "test1" },
-        { type: "string", value: "test2" },
-        { type: "string", value: "test3" }
-      ])
-    );
+      deepSearch(testData, item => item.type === "string")).toEqual([
+      { type: "string", value: "test1" },
+      { type: "string", value: "test2" },
+      { type: "string", value: "test3" }
+    ]);
```

Comment:
Looks like some formatting changes were introduced. I don't think they are right either. Cna you back these out?

Reply:
Sure! The reason I put those changes in was because the tests didn't seem to be running correctly with the expect block parentheses ordered the first way -- it was looking for the results of the function to have 'toEqual' property?

Reply:
i.e. this would pass all the 'deep search test' specs: 
```
  return {
    toEqual: () => []
  }
```

Reply:
@ktilcu Given ^, would you still like me to take those changes out?